server/services/utils/pdfFormatter.js

```diff
@@ -1,7 +1,7 @@
 const moment = require('moment')
 const { getIn, isEmpty, mergeWithRight } = require('../../utils/functionalHelpers')
 const pdfData = require('../config/pdfData')
-const { romanise } = require('../../utils/romanise')
+const romanise = require('romannumerals')
```

Comment:
odd, I would have expected this to have failed linting since the `require` is after local `require` statements